config/mfPythonDefsGEM.mk

```diff
@@ -9,6 +9,10 @@ PYTHON_LIB     ?= python${PYTHON_VERSION}
 # Python Config
 PYTHONCFLAGS = $(shell pkg-config python --cflags)
 PYTHONLIBS   = $(shell pkg-config python --libs)
+PYTHONGLIBS  = $(shell pkg-config python --glibs)
+
+IncludeDirs+=${PYTHON_INCLUDE_PREFIX}
+
```

Comment:
I was pointed to this https://github.com/cms-gem-daq-project/cmsgemos/blob/generic-amc-release-v3/config/mfPythonDefsGEM.mk because of problems during the compilation related to the file Python.h not found.

Reply:
Actually this has to be changed to python compilation flags because that has been changed towards the develop...
https://github.com/cms-gem-daq-project/cmsgemos/blob/develop/config/mfPythonDefsGEM.mk

Reply:
This *should* be taken care of by the changes to the `gemcalibration/Makefile`, which was why I was curious why this change was introduced.

```
UserCFlags  +=$(PYTHONCFLAGS)
UserCCFlags +=$(PYTHONCFLAGS)
```